### PR TITLE
HTS: Add HTS REST API tests to monitor

### LINKED ISSUE #1150: HTS: Add to monitor
**Problem**
We added HTS APIs in 0.21 but don't currently monitor them.

**Solution**
Add the new HTS REST APIs to the monitoring API and dashboard.

**Alternatives**

**Additional Context**

### PR DESCRIPTION
**Pull request by xin-hedera**
**Detailed description**:
- Add HTS REST API tests to monitor
- Add option to enable / disable tests for a specific resource
- Fix incorrect timestamp query parameter handling in token balances API

**Which issue(s) this PR fixes**:
Fixes #1150 

**Special notes for your reviewer**:

**Checklist**
- [x] Documentation added
- [x] Tests updated

### CODE REVIEW

File: hedera-mirror-rest/monitoring/README.md [hunk 1]
```diff
@@ -51,38 +51,56 @@ nano config/serverlist.json // Insert the mirror node deployments you want to mo
...
-- `intervalMultiplier` for all resources. The tests for a resource will run every `interval * intervalMultiplier` seconds.
-   For `stateproof`, it defaults to 10, so the tests run at a lower frequency to reduce cost
-- `limit` threshold for `account`, `balance`, `transatcion`, and `topic` can be adjusted independently if needed, e.g.,
-  for environments with lower traffic volume
+- set `enabled` to `true` or `false` to enable / disable tests for a resource
+- `freshnessThreshold` (in seconds) for `balance`, `transaction`, `topic`, and `token` (balances of a token) can be
+   adjusted independently if needed. Set to 0 to disable freshness check for a resource
+- `intervalMultiplier` for all resources. The tests for a resource will run every `interval * intervalMultiplier`
+   seconds. For `stateproof`, it defaults to 10, so the tests run at a lower frequency to reduce cost
+- `limit` threshold for `account`, `balance`, `transatcion`, `topic`, and `token` can be adjusted independently if
```
Comment (steven-sheehy) [inline hedera-mirror-rest/monitoring/README.md:59]: `transaction`

File: hedera-mirror-rest/monitoring/monitor_apis/monitor_tests.js [hunk 2]
```diff
@@ -46,7 +47,8 @@ const counters = {};
 const runTests = (name, address) => {
   const counter = name in counters ? counters[name] : 0;
   const skippedResource = [];
-  const testModules = allTestModules.filter((testModule) => {
+  const enabledTestModules = allTestModules.filter((testModule) => config[testModule.resource].enabled);
```
Comment (steven-sheehy) [inline hedera-mirror-rest/monitoring/monitor_apis/monitor_tests.js:50]: Should check if enabled flag is present as well to not break existing configs.

**Review by steven-sheehy — CHANGES_REQUESTED** [review]

File: hedera-mirror-rest/monitoring/README.md [hunk 3]
```diff
@@ -51,38 +51,56 @@ nano config/serverlist.json // Insert the mirror node deployments you want to mo
...
 To customize per-resource configuration:
 
-- `freshnessThreshold` (in seconds) for `balance`, `transaction`, and `topic` can be adjusted independently if needed.
-   Set to 0 to disable freshness check for a resource
-- `intervalMultiplier` for all resources. The tests for a resource will run every `interval * intervalMultiplier` seconds.
-   For `stateproof`, it defaults to 10, so the tests run at a lower frequency to reduce cost
-- `limit` threshold for `account`, `balance`, `transatcion`, and `topic` can be adjusted independently if needed, e.g.,
-  for environments with lower traffic volume
+- set `enabled` to `true` or `false` to enable / disable tests for a resource
+- `freshnessThreshold` (in seconds) for `balance`, `transaction`, `topic`, and `token` (balances of a token) can be
```
[the remark this answers is not in the document]
Comment (xin-hedera) [reply hedera-mirror-rest/monitoring/README.md:55]: nope, `account` doesn't have a freshness check

**Review by steven-sheehy — APPROVED** [review]
LGTM

**Review by Nana-EC — APPROVED** [review]
LGTM

**Pull request merged**